### PR TITLE
with modwsgi config change to ini new relic file

### PR DESCRIPTION
**Pull request by vccabral**
@sleitner @imuchnik @marcesher 

I need a testing method to ensure the ini file is parse-able and works in our environments to see if we, indeed do, need to follow marks additional instructions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ Ansible scripts for installing php or python [New Relic](http://www.newrelic.com
 The project is configured to require an encrypted license.yml file -- containing the New Relic license key -- that gets opened with a password. To create the encrypted license.yml file follow [these instructions](http://docs.ansible.com/ansible/playbooks_vault.html#creating-encrypted-files), entering the password above when prompted, and creating a file containing the following.
 
 ```
+---
```
Comment (vccabral) [inline README.md:13]: For clarity, these lines start every yml file. The read me just included the vars.

**Comment by sleitner** [comment]
@vccabral Did you get a chance to test it locally? This LGTM.

**Comment by vccabral** [comment]
@sleitner I ran the provision steps locally and can confirm the line is in the file where I expect. Also in the newrelic admin portal i see 2 test boxes but I do not see any errors that have ever been reported to those machines.

**Comment by vccabral** [comment]
@sleitner tested and we are approved. Do you have merge access?

**Comment by sleitner** [comment]
:+1:

**Pull request merged**